src/main/java/edu/kis/powp/jobs2d/command/ComplexCommand.java

```diff
@@ -0,0 +1,25 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.Iterator;
+import java.util.List;
+
+public class ComplexCommand implements ICompoundCommand {
+
+    private final List<DriverCommand> listOfCommands;
+
+    public ComplexCommand(List<DriverCommand> listOfCommands) {
```

Comment:
please add 'name' field and return it in toString - would make work with CommandManager easier.